plugins/DuskVerb/tools/tuner/ab_fdn_vs_accuratehall.py

```
@@ -0,0 +1,98 @@
+        x, sr = sf.read(f); sf.write(f, x * g, sr)
+
+
+def lexdir(adir, prefix, dst):
+    os.makedirs(dst, exist_ok=True)
+    for s in STIM:
+        src = f"{adir}/{prefix}_{s}.wav"
+        if os.path.exists(src): shutil.copy(src, f"{dst}/anchor_{s}.wav")
+
+
+def run_fc(dvdir, lex, name):
+    r = subprocess.run([sys.executable, FC, dvdir, lex, "--name", name, "--json"],
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)